packages/avs/src/http/body.rs

```diff
@@ -0,0 +1,68 @@
+//! HTTP body types
```

Comment:
IIRC much of this code was forked from https://github.com/yoshuawuyts/wstd which is under MIT / Apache-2.0 license. There is no attribution required, but I still find it nice to add some comment pointing to the original source (and maybe the PR that didn't make it) to give context... also for later comparisons if we want to add new features they added